@ecomplus/widget-product/src/index.js

```diff
@@ -54,7 +54,8 @@ export default (options = {}, elId = 'product') => {
             : strHasQuantitySelector ? Boolean(strHasQuantitySelector.trim()) : false,
           lowQuantityToWarn,
           maxVariationOptionsBtns,
-          isSSR
+          isSSR,
+          showPromotionalTimer: true  // storefront.showPromotionalTimer ???
```

Comment:
Na verdade seria algo como:

```suggestion
          hasPromotionTimer: options.hasPromotionTimer
```

Mas até que isso poderia ser feito em outra PR, junto com a edição da configuração respectiva dos fields pra esse widget no CMS

Reply:
Resolvido